frontend/src/index.js

```diff
@@ -1,16 +1,17 @@
 // Import libraries
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
 
 // Import local files
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
-  <React.StrictMode>
+  <BrowserRouter>
```

Comment:
I think you should actually keep the `<React.StrictMode>` here and put the Router in App.js